### ISSUE TITLE
Replace simple calibration file with a config.py file

### ISSUE THREAD

**Issue by kaufmanno** [UMONS-GFA/ardas#69]
The config.py file should contain all data to configure the ardas and transform measured frequencies coming from the sensors in readable data.

**Comment by kaufmanno** [comment]
Should the config.py be different from the settings.py, or would it be better to merge them in a single file? @bastinc

**Comment by kaufmanno** [comment]
I changed the approach a little bit: 
1.  I wrote sensor_tools.py to provide a `FMSensor` class and methods to process incoming frequencies (i.e. calibration using a polynomial expression, running average...). More methods could be added in the future to fit other use cases.
2.  This module should be imported in a python file (see Commodore64_sensors_cilibration.py for an example) where objects corresponding to actual sensors are created from the `FMSensor' class and saved in files named sensor_####.ssr where #### is the sensor identifier (usually an integer)
3.  A reference to these sensor_####.ssr files is added in the settings.py::# ArDAS sensors objects connected to the ArDAS channels
SENSORS = (st.load_sensor('0001'), st.load_sensor('0002'), st.load_sensor('0003'), st.load_sensor('0004'))::
